src/config/kernel.md

```diff
@@ -21,28 +21,40 @@ modules.
 
 ## Removing old kernels
 
-To list kernels that can be purged
+When updating the kernel, old versions are left behind in case it is neccessary to roll back to an older version.
+Over time, many old kernel version accumulate and make updating dmks modules take a long time.
+Thus, it may be advisable to clean old kernels from time to time.
+
+Removing old kernels is done using the [vkpurge(8)](https://man.voidlinux.org/vkpurge.8) utility.
+`vkuprge` comes pre-installed on every Void Linux system.
+This utility runs the necessary hooks when removing old kernels.
+
+To list kernels that can be purged:
 
 ```
 $ vkpurge list
 3.8.2_1
 ```
 
-To remove a specific kernel version like `3.8.2_1`
+To remove a specific kernel version like `3.8.2_1`:
 
 ```
 # vkpurge rm 3.8.2_1
 ```
 
 To remove `all` kernels except the latest kernel of each series and the kernel
-that is currently booted
+that is currently booted:
 
 ```
 # vkpurge rm all
 ```
 
 ## Kernel modules
 
+The Linux kernel has modular components which are called modules.
+Kernel modules are typically drivers for devices or filesystems.
```

Comment:
I don't think these first two lines are necessary for the target audience, and especially not the first one. This can be really shortened or removed.

Reply:
I agree that the first sentence is superfluous.